sbws/lib/v3bwfile.py

```diff
@@ -827,17 +848,25 @@ def bw_torflow_scale(bw_lines, desc_bw_obs_type=TORFLOW_OBS_MEAN,
             # See https://trac.torproject.org/projects/tor/ticket/8494
             # just applying the formula above:
             desc_bw = min(desc_bw_obs, l.desc_bw_bur, l.desc_bw_avg)
+            if l.cons_is_unmeasured:
+                min_desc_cons_bw = desc_bw
+            # If the relay is measured, use the minimum between the descriptors
+            # bandwidth and the consensus bandwidth
```

Comment:
This change from Torflow fixes a bug, so we should say *why* we are doing it:
```suggestion
            # bandwidth and the consensus bandwidth, so that
            # MaxAdvertisedBandwidth limits the consensus weight
```